lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -1073,11 +1078,16 @@ def build_cmd_set(self, playval, existing, k):
             [proposed.append(ctx) for ctx in ref[k].get('context', [])]
             proposed.append(cmd)
 
-    def get_proposed(self):
+    def get_proposed(self, save_context=None):
```

Comment:
ditto for this one. Our methods tend to get option heavy.

Reply:
I will actually remove this.  Cruft from a former design that I abandoned :)